src/main/resources/META-INF/reaktivity/amqp.idl

```diff
@@ -132,6 +138,7 @@ scope amqp
             uint8 flags;   // set of AmqpTransferFlag
             AmqpAnnotation[] annotations;
             AmqpMessageProperty[] properties;
+            AmqpApplicationProperty[] applicationProperties;
```

Comment:
Perhaps the message properties could be a `list` instead of an `array` of `union`?
What would be the implications?

Reply:
I agree. In fact, `list` is more accurate since the definition of `properties` in `amqp` documentation is a `list` and therefore the order matters (the current implementation with `array` of `union` does not restrict the ordering.)

Should I change it to `list` like the following?
```
list<uint32, uint32> AmqpProperties
{
    AmqpMessageId messageId;
    AmqpBinary userId;
    string8 to;
    string8 subject;
    string8 replyTo;
    AmqpMessageId correlationId;
    string8 contentType;
    string8 contentEncoding;
    int64 absoluteExpiryTime;
    int64 creationTime;
    string8 groupId;
    int32 groupSequence;
    string8 replyToGroupId;
}
```

Reply:
Yes, that looks great! 👍 

Note: the current implementation of `array` of `union` also doesn't prevent duplicates which the `list` approach also resolves.